scene/main/canvas_item.h

```diff
@@ -207,6 +211,13 @@ class CanvasItem : public Node {
 	virtual void set_light_mask(int p_light_mask);
 	int get_light_mask() const;
 
+	void set_layer_mask(int p_layer_mask);
```

Comment:
layer should be uint32_t, not int.

Reply:
I was hoping this would come up, is there any reason why other masks are some int some not? I wasn't 100% sure on the interactions with gdscript pushing only ints.

Reply:
I have changed these types and related usage in 3221c23. This now matches `VisualInstance3D` better.

Reply:
They should probably changed too, it was originally like this because Variant contained int32_t, but now that it contains int64_t there is not much of a point in artificially limiting to int.